package.json

```diff
@@ -105,79 +105,76 @@
     "@deque/dot": "^1.1.5",
-    "act-rules.github.io": "github:act-rules/act-rules.github.io#a5d7336e020fd04fd429f94c651102f8ab255f01",
-    "aria-practices": "github:w3c/aria-practices#f7de7ec3a53534018237f24cb9e610f26c30c367",
-    "aria-query": "^3.0.0",
-    "browser-driver-manager": "1.0.3",
-    "chai": "~4.2.0",
-    "chalk": "^4.1.0",
+    "act-rules.github.io": "github:act-rules/act-rules.github.io#a759e99f",
+    "aria-practices": "github:w3c/aria-practices#edbf534",
+    "aria-query": "^5.0.0",
+    "browser-driver-manager": "1.0.4",
+    "chai": "~4.3.6",
+    "chalk": "^4.x",
     "chromedriver": "latest",
     "clone": "^2.1.2",
-    "conventional-commits-parser": "^3.1.0",
-    "css-selector-parser": "^1.3.0",
-    "emoji-regex": "8.0.0",
-    "es6-promise": "^4.2.6",
-    "esbuild": "^0.7.19",
-    "eslint": "^7.27.0",
-    "eslint-config-prettier": "^6.2.0",
+    "conventional-commits-parser": "^3.2.4",
+    "core-js": "^3.23.4",
+    "css-selector-parser": "^1.4.1",
+    "emoji-regex": "10.1.0",
+    "es6-promise": "^4.2.8",
+    "esbuild": "^0.14.49",
+    "eslint": "^8.19.0",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-mocha-no-only": "^1.1.1",
-    "execa": "^2.0.2",
+    "execa": "5.x",
     "globby": "^10.0.0",
-    "grunt": "^1.0.3",
+    "grunt": "^1.5.3",
     "grunt-babel": "^8.0.0",
     "grunt-bytesize": "^0.2.0",
-    "grunt-contrib-clean": "^2.0.0",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-uglify": "^4.0.0",
+    "grunt-contrib-clean": "^2.0.1",
+    "grunt-contrib-concat": "^2.1.0",
+    "grunt-contrib-uglify": "^5.2.2",
     "grunt-contrib-watch": "^1.1.0",
-    "html-entities": "^1.2.0",
-    "http-server": "^14.1.0",
-    "husky": "^3.1.0",
-    "jquery": "^3.0.0",
+    "html-entities": "1.x",
+    "http-server": "^14.1.1",
+    "husky": "^8.0.1",
+    "jquery": "^3.6.0",
     "jsdoc": "^3.6.10",
-    "jsdom": "^16.3.0",
-    "karma": "^6.3.16",
+    "jsdom": "^20.0.0",
+    "karma": "^6.4.0",
     "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-firefox-launcher": "^2.1.0",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-firefox-launcher": "^2.1.2",
     "karma-ie-launcher": "^1.0.0",
     "karma-mocha": "^2.0.1",
     "karma-sinon": "^1.0.5",
-    "karma-spec-reporter": "0.0.33",
-    "lint-staged": "^11.2.6",
-    "make-dir": "^3.0.0",
-    "markdown-table": "^1.1.2",
-    "memoizee": "^0.4.14",
+    "karma-spec-reporter": "0.0.34",
+    "lint-staged": "^13.0.3",
+    "make-dir": "^3.1.0",
+    "markdown-table": "2.x",
+    "memoizee": "^0.4.15",
     "minami": "^1.2.3",
-    "mocha": "^9.2.0",
-    "node-notifier": "^8.0.1",
+    "mocha": "9.x",
+    "node-notifier": "^10.0.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^1.17.1",
+    "prettier": "^2.7.1",
     "proxyquire": "^2.1.3",
     "revalidator": "~0.3.1",
-    "selenium-webdriver": "~3.6.0",
-    "sinon": "^7.5.0",
+    "selenium-webdriver": "3.x",
```

Comment:
I think we should upgrade this to use `4.3.0`

Reply:
I'll take a look. I remember now that you shared an update about a breaking change with that package; maybe that was my issue with the newer 4.3.1.